packages/react-data-grid/src/Row.js

```diff
@@ -49,17 +52,16 @@ class Row extends React.Component {
     onDragEnter({ overRowIdx: idx });
   };
 
-  handleDrop= (e) => {
+  handleDrop = (e) => {
     e.preventDefault();
   };
 
   getCell = (column, i) => {
     const CellRenderer = this.props.cellRenderer;
-    const { idx, cellMetaData } = this.props;
+    const { idx, cellMetaData, isScrolling, row, isSelected, scrollLeft, lastFrozenColumnIndex } = this.props;
     const { key, formatter } = column;
-    const baseCellProps = { key: `${key}-${idx}`, idx: i, rowIdx: idx, height: this.getRowHeight(), column, cellMetaData };
+    const baseCellProps = { key: `${key}-${idx}`, idx: column.idx, rowIdx: idx, height: this.getRowHeight(), column, cellMetaData };
```

Comment:
@amanmahajan7, it looks like the cause of https://github.com/adazzle/react-data-grid/issues/1343 may be that the change from `i` to `column.idx` was not fully done in `getCell`. It looks like `i` can just be removed as a second parameter here and then I can change `key || i` to `key || column.idx`.

Before I do that though, is there any chance that would break frozen columns or some other new feature?